buildahimage/README.md

```diff
@@ -11,7 +11,7 @@ resulting containers can run safely with privileges within the container.  The c
 using the latest Fedora and then Buildah is installed into them:
 
   * quay.io/buildah/buildahimage - This image is built using the latest stable version of Buildah in a Fedora based container.  Built with buildahimage/stable/Dockerfile.
-  * quay.io/buildah/buildahimagegit - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Containers built with this imsage have a Buildah development environment set up in the directory /root/buildah.  Built with buildahimage/git/Dockerfile.
+  * quay.io/buildah/buildahimageupstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with buildahimage/upstream/Dockerfile.
```

Comment:
why `buildahimageupstream`? that is a clunky name and one that i would forget and have to search everytime.
whereas `quay.io/buildah/buildah` i might would guess without even searching

Reply:
We've:

o buildahimage - Latest stable version

o buildahimageupstream - Latest version from upstream GitHub

o buildahimagetesting - Latest version from updates testing, at times the same as stable.

pullable (or will be pullable, now it's buildahimagegit), not  with:

buildah pull docker://quay.io/buildah/buildahimage
buildah pull docker://quay.io/buildah/buildahimageupstream
buildah pull docker://quay.io/buildah/buildahimagetesting

Happy to take naming suggestions, but I'm hoping to get a blog together for next week and would like to cement it by then.

Thinking a bit, would removing the "buildahimage" from the above work?  I think I'd still keep the Dockerfiles as is naming wise.  Something like:

buildah pull docker://quay.io/buildah/stable
buildah pull docker://quay.io/buildah/upstream
buildah pull docker://quay.io/buildah/testing

I think I like that better sitting back and pondering.

Reply:
ooOOoo short, simple and expressive. :-)

Reply:
Cool, then once homu gets over itself and merges these changes, I'll make the adjustments on quay.io to make it so.  Thanks for the discussion!